index.html

```diff
@@ -72,5 +80,6 @@ <h2 class="section-header">Connect</h2>
             <li><a href="https://github.com/temuka-code" target="_blank">Visit my GitHub!</a></li>
         </ul>
     </section>   
+    <script src="js/index.js"></script>
 </body>
 </html>
```

Comment:
No huge deal, but the reason there is this red symbol at the end of the files is that GitHub is "yelling at you" to leave one extra empty line at the end of each file, which is standard practice. Just fyi that it'll go away for your next PR if you do that.

<img width="26" alt="Screenshot 2025-01-26 at 12 28 26 PM" src="https://github.com/user-attachments/assets/d4c3a090-a5e4-48e6-9c00-df791537a17d" />